README.markdown

```diff
@@ -151,6 +151,14 @@ The directory the puppet agent should be installed to. This is only applicable f
   install_dir => 'D:\Program Files\Puppet Labs'
 ```
 
+##### `install_options`
+
+An array of additional options to pass when installing a package. These options are package-specific, and should be documented by the software vendor. One commonly implemented option is `INSTALLDIR` Each option in the array can either be a string or a hash, where each key and value pair are interpreted in a provider specific way. Each option will automatically be quoted when passed to the install command. With Windows packages, note that file paths in an install option must use backslashes. (Since install options are passed directly to the installation command, forward slashes won't be automatically converted like they are in `file` resources.) Note also that backslashes in double-quoted strings _must_ be escaped and backslashes in single-quoted strings _can_ be escaped.
```

Comment:
Since we control the package, shouldn't we just point to where these options are defined (rather than say they "documented by the software vendor")?